safe/report/extractors/aggregate_result.py

```diff
@@ -99,7 +99,7 @@ def aggregation_result_extractor(impact_report, component_metadata):
         total_affected_value = format_number(
             feat[total_field_index],
             enable_rounding=is_rounded)
-        if total_affected_value == 0:
+        if total_affected_value == '0':
```

Comment:
The value now always returned by format_number, so I think it's always a string (with decimal and thousand separator in debug mode)